tests/integ/modin/resample/test_resample_asfreq.py

```diff
@@ -56,3 +69,5 @@ def test_asfreq_negative():
         snow_df.asfreq(freq="5s", method="ffill", normalize=True)
     with pytest.raises(NotImplementedError):
         snow_df.asfreq(freq="5s", method="ffill", fill_value=2)
```

Comment:
Why are we testing `DataFrame.asfreq` here when the test file is `test_resample_asfreq`?

Reply:
`DataFrame/Series.asfreq` end up just calling the `resample` implementation. I didn't realize at that time the `Resampler` class also has an `asfreq` so this PR just adds that. That's why all 3 are tested in the same file under the resample folder.